src/common/io.py

```diff
@@ -1422,14 +1439,42 @@ def _get_description(self):
     Property for accessing the description vector.
     """)
 
-    def get_variable_data(self, name):
+    def _map_index_to_data_properties(self, variable_name: str) -> tuple[int, int, int]:
+        """ Returns the data matrix ID, index and factor for given variable name. """
+
+        variable_index = self.get_variable_index(variable_name)
+        data_mat = self._dataInfo[0][variable_index]
+
+        data_index = self._dataInfo[1][variable_index]
+        data_index, factor = self._correct_index_for_alias(data_index)
+
+        if data_mat == 0:
+            # Take into account that the 'Time' variable has data matrix index 0
+            # and that 'time' is called 'Time' in Dymola results
```

Comment:
Dymola results = ResultHandler types containing "Dymola" in the name or the actual tool Dymola?

Reply:
I actually don't know, this comment was here in the code before I started working on it. Perhaps @chria can shed some light on this.

Reply:
Actual tool. But I think we can just update the comment to say that we take into account that the "time" variable can be both named "time" and "Time"